HW-05/hw05(1).py

```diff
@@ -0,0 +1,17 @@
+text = input('Enter your text: ')
+for item in text:
+    print(item)
+    if item.isdigit():
+        print('This is digit.')
+        if int(item) % 2 == 0:
+            print('This is even.')
+        else:
+            print('This is add.')
+    elif item.isalnum():
+        print('This is letter.')
+        if item.istitle():
```

Comment:
`istitle` checks if a word starts with a capital letter. According to the task it can be used, but there is also the function `isupper` that can check if a character is uppercase and `islower` for lowercase